scripts/abbreviations/LocalizeAbbreviations.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,272 @@
+// !$*UTF8*$!
```

Comment:
It's not really reusable as is so just adding a ref to the script in the MapboxNavigation project as you mention makes sense. However, SwiftCLI doesn't provide a project file and relies on SPM. We need to checkout, generate project and build with Carthage separately. It would not be convenient to force [this flow](https://github.com/mapbox/mapbox-navigation-ios/pull/254/commits/ad8e1c9d2238a95cd6824a68fd17bb89f4853490#diff-a51af85c97fc8578019ea23f6f3a0319R4) every time we use Carthage in this project.